components/org.wso2.carbon.identity.conditional.auth.functions.analytics/pom.xml

```diff
@@ -109,6 +109,12 @@
             <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
             <artifactId>httpasyncclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
+            <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
+            <version>${identity.organization.management.core.version}</version>
```

Comment:
Why we need to define the version in the component pom file?

Reply:
removed the versions from component pom files.